### PR TITLE
feat: Version info

### PR DESCRIPTION
**Pull request by yahu1031**
Added `version` command.

```console
PS F:\Projects\serverpod\tools\serverpod_cli> dart .\bin\serverpod_cli.dart version
WARNING! Windows is not officially supported yet. Things may or may not work as expected.

Serverpod version: 0.9.10
```

## Pre-launch Checklist

- [x] I read the [Contribute](https://docs.serverpod.dev/contribute) page and followed the process outlined there for submitting PRs.
- [x] I read and followed the [Dart Style Guide](https://dart.dev/guides/language/effective-dart/style) and formatted the code with [dart format](https://dart.dev/tools/dart-format).
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`), and made sure that the documentation follows the same style as other Serverpod documentation. I checked spelling and grammar.
- [ ] I added new tests to check the change I am making.
- [x] All existing and new tests are passing.
- [x] Any breaking changes are documented below.

## Breaking changes

- No breaking changes were made
- Made an enhancement.

### CODE REVIEW

File: tools/serverpod_cli/pubspec.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ dependencies:
   recase: ^4.0.0
   super_string: ^1.0.3
   watcher: ^1.0.0
-  analyzer: ^3.3.1
+  analyzer: ^4.0.0
```
Comment (vlidholt) [inline tools/serverpod_cli/pubspec.yaml:22]: The analyzer package shouldn't be updated in this PR.

File: tools/serverpod_cli/bin/util/version.dart [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import '../generated/version.dart';
+import 'print.dart';
+
+/// Get the current version of the serverpod.
+void getVersion() => printww('Serverpod version: $templateVersion');
```
Comment (vlidholt) [inline tools/serverpod_cli/bin/util/version.dart:5]: The naming `getVersion` implies that the method returns a value. A better name would be `printVersion`. Also, please use the `() {}` method syntax unless the method is returning a value.

File: tools/serverpod_cli/bin/serverpod_cli.dart [hunk 3]
```diff
@@ -322,7 +332,8 @@ void _printUsage(ArgParser parser) {
   print('');
   print('${Colorize('COMMANDS')..bold()}');
   print('');
-
+  _printCommandUsage(
+      cmdVersion, 'Prints the current active version of serverpod.');
```
Comment (vlidholt) [inline tools/serverpod_cli/bin/serverpod_cli.dart:336]: Please add a trailing comma for consistency and readability.

**Review by vlidholt — CHANGES_REQUESTED** [review]
Great addition! They will be accepted with the minor fixes mentioned above. :)

File: tools/serverpod_cli/pubspec.yaml (hunk 1 above)
Comment (yahu1031) [reply tools/serverpod_cli/pubspec.yaml:22]: Done

File: tools/serverpod_cli/bin/serverpod_cli.dart (hunk 3 above)
Comment (yahu1031) [reply tools/serverpod_cli/bin/serverpod_cli.dart:336]: Done

File: tools/serverpod_cli/bin/util/version.dart (hunk 2 above)
Comment (yahu1031) [reply tools/serverpod_cli/bin/util/version.dart:5]: Done.

**Review by vlidholt — APPROVED** [review]

**Pull request merged**